CONTRIBUTING.md

```diff
@@ -134,13 +134,13 @@ As a basic demonstration, we'll be implementing a command which prints the name
 
 #### Creating the Command
 
-Like all other commands, this command will reside in `internal/cmd`. First, we must create a directory for this command:
+Like all other commands, this command will reside in `internal`. First, we must create a directory for this command:
 
-    mkdir internal/cmd/config
+    mkdir internal/config
```

Comment:
We have a `configuration` command now, so maybe we should come up with a new example in the future